.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: '🐛 Bug Report'
+about: Did FAST not work as expected?
+title: "fix: [what is the issue?] in [where is the issue?]"
+labels: 'status:triage'
+---
+
+<!---
+Thanks for filing an issue 😄 ! Before you submit, please read the following:
+
+Search open/closed issues before submitting. Someone may have reported the same issue before.
+-->
+
+# 🐛 Bug Report
+
+<!--- Provide a general summary of the issue here -->
+
+## 🤔 Expected Behavior
+
+<!--- Tell us what should happen -->
+
+## 😯 Current Behavior
+
+<!--- Tell us what happens instead of the expected behavior -->
+<!--- If you are seeing an error, please include the full error message and stack trace -->
+<!--- If applicable, provide screenshots -->
+
+## 💁 Possible Solution
+
+<!--- Not obligatory, but suggest a fix/reason for the bug -->
+<!--- Please let us know if you'd be willing to contribute the fix; we'd be happy to work with you -->
+
+## 🔦 Context
+
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## 💻 Repro or Code Sample
```

Comment:
Thoughts on moving this just below "Bug Report"? To me, the repro steps are super important and set context for "expected behavior" and "current behavior"

Reply:
Yeah, makes sense. Give me a sec...